web-local/src/routes/(misc)/deploy/+page.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  import CancelCircleInverse from "@rilldata/web-common/components/icons/CancelCircleInverse.svelte";
+  import { EntityStatus } from "@rilldata/web-common/features/entity-management/types";
+  import OrgSelector from "@rilldata/web-common/features/project/OrgSelector.svelte";
+  import { ProjectDeployer } from "@rilldata/web-common/features/project/ProjectDeployer";
+  import { onMount } from "svelte";
+  import CTAContentContainer from "@rilldata/web-common/components/calls-to-action/CTAContentContainer.svelte";
+  import CTALayoutContainer from "@rilldata/web-common/components/calls-to-action/CTALayoutContainer.svelte";
+  import CTAHeader from "@rilldata/web-common/components/calls-to-action/CTAHeader.svelte";
+  import CTANeedHelp from "@rilldata/web-common/components/calls-to-action/CTANeedHelp.svelte";
+  import CTAMessage from "@rilldata/web-common/components/calls-to-action/CTAMessage.svelte";
+  import CTAButton from "@rilldata/web-common/components/calls-to-action/CTAButton.svelte";
+  import Spinner from "@rilldata/web-common/features/entity-management/Spinner.svelte";
+
+  const deployer = new ProjectDeployer();
+  const deployValidation = deployer.validation;
+  const deployerStatus = deployer.getStatus();
+  const promptOrgSelection = deployer.promptOrgSelection;
+
+  function onOrgSelect(org: string) {
+    promptOrgSelection.set(false);
+    return deployer.deploy(org);
+  }
+
+  function onContactUs() {
+    // TODO
+  }
+
+  onMount(() => {
+    void deployer.deploy();
+  });
+</script>
+
+<CTALayoutContainer>
+  <CTAContentContainer>
+    {#if $promptOrgSelection}
+      <OrgSelector
+        orgs={$deployValidation.data?.rillUserOrgs ?? []}
+        onSelect={onOrgSelect}
+      />
+    {:else if $deployerStatus.isLoading}
+      <div class="h-36">
+        <Spinner status={EntityStatus.Running} size="7rem" duration={725} />
+      </div>
+      <CTAHeader variant="bold">
+        Hang tight! We're deploying your project...
+      </CTAHeader>
+    {:else if $deployerStatus.error}
+      <div class="h-36">
+        <CancelCircleInverse size="7rem" className="text-gray-200" />
+      </div>
+      <CTAHeader variant="bold">Oops! An error occurred</CTAHeader>
+      <CTAMessage>{$deployerStatus.error}</CTAMessage>
+      <CTAButton variant="secondary" on:click={onContactUs}>
```

Comment:
Here too we should use `href` not `on:click` for links

Reply:
This will open pylon similar to AvatarButton. Need pylon integration to local before this is merged.

Reply:
Oh now I see. Given the event handler isn't yet implemented, we should hide the button.